app/src/main/AndroidManifest.xml

```diff
@@ -5,7 +5,7 @@
     android:versionName="1.0" >
 
     <uses-sdk
-        android:minSdkVersion="10"
+        android:minSdkVersion="11"
```

Comment:
Since we're not very interested in tablets, this cuts out all devices before Android 4.0, I believe. What are we using that requires that?